slack-base/src/main/java/com/hubspot/slack/client/models/blocks/elements/UsersMultiSelectMenuIF.java

```diff
@@ -0,0 +1,37 @@
+package com.hubspot.slack.client.models.blocks.elements;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.immutables.value.Value;
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.blocks.objects.ConfirmationDialog;
+import com.hubspot.slack.client.models.blocks.objects.Text;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface UsersMultiSelectMenuIF extends BlockElement {
+  String TYPE = "multi_users_select";
+
+  @Override
+  @Value.Derived
+  default String getType() {
+    return TYPE;
+  }
+
+  Text getPlaceholder();
+
+  String getActionId();
+
+  @JsonProperty("initial_users")
+  List<String> getInitialUserIds();
```

Comment:
This feels like it should be a `Set`

Reply:
Duplicates are allowed by Slack in this field, so I don't think `Set` would be appropriate.

Reply:
![All the Johanns!](https://user-images.githubusercontent.com/4570406/68799532-3b48d280-0650-11ea-80d4-4c8f305f244c.png)

Reply:
loooooool ok I stand corrected